static/app/components/charts/simpleTableChart.tsx

```diff
@@ -59,45 +71,36 @@ class SimpleTableChart extends Component<Props> {
     });
   }
 
-  render() {
-    const {
-      className,
-      loading,
-      fields,
-      metadata,
-      data,
-      title,
-      fieldHeaderMap,
-      stickyHeaders,
-    } = this.props;
-    const meta = metadata ?? {};
-    const columns = decodeColumnOrder(fields.map(field => ({field})));
-    return (
-      <Fragment>
-        {title && <h4>{title}</h4>}
-        <StyledPanelTable
-          className={className}
-          isLoading={loading}
-          headers={columns.map((column, index) => {
-            const align = fieldAlignment(column.name, column.type, meta);
-            const header = fieldHeaderMap?.[column.key] ?? column.name;
-            return (
-              <HeadCell key={index} align={align}>
-                <Tooltip title={header}>
-                  <StyledTruncate value={header} maxLength={30} expandable={false} />
-                </Tooltip>
-              </HeadCell>
-            );
-          })}
-          isEmpty={!data?.length}
-          stickyHeaders={stickyHeaders}
-          disablePadding
-        >
-          {data?.map((row, index) => this.renderRow(index, row, meta, columns))}
-        </StyledPanelTable>
-      </Fragment>
-    );
-  }
+  const meta = metadata ?? {};
```

Comment:
Feel free to just inline metadata ?? {} into the render, it looks like it's only used there